images/kubekins-e2e-v2/variants.yaml

```diff
@@ -31,6 +31,12 @@ variants:
     K8S_RELEASE: stable
     BAZEL_VERSION: 3.4.1
     OLD_BAZEL_VERSION: 2.2.0
+  '1.32':
+    CONFIG: '1.32'
+    GO_VERSION: 1.23.2
```

Comment:
Can you update this and all other occurrences of 1.23.2 to 1.23.3?

Reply:
updated!